.github/tests/setup.sh

```diff
@@ -132,6 +137,12 @@ setup_site() {
 		STATUS_ERROR
 		exit 1
 	fi
+    STATUS_TITLE "Setup required Directories"
```

Comment:
When can also remove these below for the same reason

Reply:
idem